### DISCUSSION TITLE
Does this PnP module support SharePoint SE?

### DISCUSSION THREAD

**Discussion by wspelt** [pnp/powershell#3147] [Q&A] [answered]
Hi All,

For a customer we are about to migrate to SharePoint 2019.
However, we are learning that SharePoint Server Subscription Edition (SE) is recommended by various sources as it is no longer associated with a 'version number', but receives continuous updates.
This sounds interesting. However, we also look at how we can develop against it.
We are now deploying templates with PnP PowerShell provisioning and also use the PnPJS library, which both work with 2019.
We are looking for clarification on PnP support for SharePoint SE.
**So the question is: does this PnP module support SharePoint SE?**

And if the answer is no, the next question of course would be:
Do we have to provision our templates on-premises in the old-fashioned way with a wsp package and make features with standard SharePoint XML and also use the standard Microsoft.SharePoint snap-in for PowerShell?

An answer would be very much appreciated!
We already need to decide on this next week, since our customer want's to start installing either version of SP 2019 or SPSE on multiple farms/servers.

**Comment by r998877** [comment]
Same question here. Why is nobody answering?

**Reply by wspelt** [reply]
We are already using the PnP SharePoint 2019 PowerShell module on SharePoint SE and also the PnPJS library and so far everything seems to work fine. But with the ongoing update model in SE my guess is this is not supported at this moment by the PnP team, since I cannot find any information about this and there is no official PnP SharePoint SE module. Hopefully it get's supported.

@erwinvanhunen Could you maybe clarify this question?
Is the SharePoint SE on-premises model also new for the PnP team?

**Reply by gautamdsheth** [reply, marked as answer]
This module is not supported any version of SharePoint on-prem. Only SharePoint online is supported.
